operations/cp.go

```diff
@@ -0,0 +1,41 @@
+package operations
+
+import (
+	"io"
+)
+
+func (do *DualOperator) Copy(src, dst string) {
```

Comment:
How about introducing a ResultChannel like https://github.com/beyondstorage/beyond-ctl/pull/26 ?

Reply:
Result contains `targetObject` and `Error`?

Reply:
For `copy` here, I think `error` is enough.

```go
type Result {
    err error
}
```

Reply:
return `err` and a `Result` only have `Err` field looks weird.